okhttp-apache/src/test/java/com/squareup/okhttp/apache/OkApacheClientTest.java

```diff
@@ -260,6 +261,16 @@
     assertEquals(text, EntityUtils.toString(entity, UTF_8));
   }
 
+  @Test public void httpHost() throws Exception {
+    server.enqueue(new MockResponse());
+    HttpHost host = new HttpHost(server.getHostName(), server.getPort());
+    HttpGet get = new HttpGet(server.getUrl("/test").getFile());
```

Comment:
I think for this test to be right you need to pass `"/test"` without the `server.getUrl()` call.